src/configuration_server.py

```diff
@@ -881,14 +881,21 @@ def get_media_changer(self, ticket):
         :arg ticket: request containing "get_media_changer" work
         """
         #__pychecker__ = "unusednames=ticket"
-
+        
         movers = self.get_movers_internal(ticket)
         ret = ''
         for m in movers:
             mv_name = m['mover']
             ret =  self.get_dict_entry(mv_name).get('media_changer','')
             if ret:
-                break
+                # handle TFF1-AB variety
+                mc = self.get_dict_entry(ret)
+                if mc.get('remote_media_changer'):
+                    ret = mc.get('remote_media_changer')
+                    if ret:
+                        break
+                else:
+                    break
```

Comment:
838-898 are equivalent to simply:
```
if mc.get('remote_media_changer'):
    ret = mc.get('remote_media_changer')
break
```
Correct? I think that is more clear

Reply:
OK